test/dynamo/test_export_mutations.py

```diff
@@ -13,7 +14,8 @@ def check_failure_on_export(self, mod, *args):
             torch._dynamo.export(mod)(*args)
 
     def check_same_with_export(self, mod, arg):
-        real_result = mod(arg)
+        eager_mod = copy.deepcopy(mod)
```

Comment:
Is deep copying usually how such things are tested? IMO it's better to take the module class instead, and just create the module instance twice.

Reply:
Yeah good idea I can do that